src/main/java/com/cloudbees/jenkins/GitHubCommitNotifier.java

```diff
@@ -34,9 +42,16 @@
  */
 public class GitHubCommitNotifier extends Notifier {
 
+    private ExpandableMessage statusMessage = new ExpandableMessage("");
```

Comment:
Provide some default messages in UI. When use adds step it should have some message so it can change it if he doesn't like it.